### ISSUE TITLE
Make Telegram code selectable in the message received by the bot

### ISSUE THREAD

**Issue by jandr** [nextcloud/twofactor_gateway#269]
If the bot were to send the code enclosed in backticks, the Telegram app allows copying that specific part of the message, which can be helpful when using the Telegram bot 2FA from mobile devices.

**Comment by ChristophWurst** [comment]
Makes sense. This needs a little refactoring, however as we currently use the very same string for all gateways, so modifications in https://github.com/nextcloud/twofactor_gateway/blob/master/lib/Service/Gateway/Telegram/Gateway.php aren't easily possible.
